es6-template-literals/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Javascript ES6 - Template Literals</title>
+  </head>
+  <body>
+    <script>
+      const number = 9200;
+      const street = `Irvine Center Drive`;
+      const address = `${number}` + ` ` + `${street}`;
+
+      console.log(`LearningFuze:`, `${address}`);
+
+      const firstName = `Uzair`;
+      const lastName = `Ashraf`;
+      const age = 25;
+
+      const bio =
+        `My name is ` +
+        `${firstName}` +
+        ` ` +
+        `${lastName}` +
+        ` and I am ` +
+        `${age}` +
+        ` years old.`;
+
+      console.log(`Bio:`, `${bio}`);
+
+      const fruit = `apple`;
+      const item = `pen`;
+
+      const applePen = fruit + item;
+
+      console.log(`I have a`, `${item}`);
+      console.log(`I have an`, `${fruit}`);
+      console.log(`${applePen}`);
```

Comment:
You shouldn't be replacing all of your strings with backticks. Keep them as strings. You are only turning the lines of code that are doing string concatenation to template literals